src/LaravelEditorJsServiceProvider.php

```diff
@@ -38,5 +39,15 @@ public function boot()
                 __DIR__ . '/../resources/views' => $this->app->resourcePath('views/vendor/laravel_editorjs'),
             ], 'laravel_editorjs-views');
         }
+
+        /**
+         * Blade directive to render editor.js blocks.
+         *
+         * @param mixed $blocks The blocks of content to be rendered.
+         * @return string Rendered HTML content.
+         */
+        Blade::directive('render', function ($blocks) {
```

Comment:
Could you please change this name to include the "editorjs" prefix or something relevant? It might create conflicts with other names.

Thank you!